tests/unit/pages/Editor.spec.js

```diff
@@ -171,22 +171,17 @@ describe("Editor.vue", () => {
       },
```

Comment:
any reason you didn't delete the imported `dummyItems` here? 
Also, there is a `dummyItemsCopy` being created and used here - what about that?